### PR TITLE
Get excluded

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/ssb-tribes/issues/80

Based on https://github.com/ssbc/ssb-tribes/pull/82

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -149,6 +149,20 @@ function init (ssb, config) {
     })
   )
 
+  pull(
+    listen.excludeMember(ssb),
+    pull.drain((msg) => {
+      const excludes = msg.value.content.excludes
+      const groupId = msg.value.content.recps[0]
```
Comment (mixmix) [inline index.js:156]:
```suggestion
      const { excludes, recps: [groupId] } = msg.value.content
```
is it readable? I dunno...

File: test/api/exclude-members.test.js [hunk 2]
```diff
@@ -25,6 +25,10 @@ test('tribes.excludeMembers', async t => {
     )
 
     await p(replicate)({ from: kaitiaki, to: newPerson, live: false, name })
+    await p(setTimeout)(100)
+
+    const newPersonGotGroup = await p(newPerson.tribes.get)(groupId)
+    t.false(!!newPersonGotGroup.excluded, 'new person is not excluded yet')
```
Comment (mixmix) [inline test/api/exclude-members.test.js:31]: this is confusing to read to me, and `!!` hides when something came back as e.g. `undefined` (coerces it to `false`)
Can you just do:
```suggestion
    t.equal(newPersonGotGroup.excluded, false, 'new person is not excluded yet')
```

File: test/api/exclude-members.test.js [hunk 3]
```diff
@@ -43,6 +47,14 @@ test('tribes.excludeMembers', async t => {
 
     const authors = await p(kaitiaki.tribes.listAuthors)(groupId)
     t.deepEqual(authors, [kaitiaki.id], 'kaitiaki removed the others from the group')
+
+    await p(replicate)({ from: kaitiaki, to: newPerson, live: false, name })
+
+    const newPersonExcludedGroup = await p(newPerson.tribes.get)(groupId)
+    t.true(!!newPersonExcludedGroup.excluded, 'new person is excluded now')
```
Comment (mixmix) [inline test/api/exclude-members.test.js:54]:
```suggestion
    t.true(newPersonExcludedGroup.excluded, 'new person is excluded now')
```

**Review by mixmix — APPROVED** [review]

**Comment by mixmix** [comment]
# NICE :pineapple: :baby_chick: :1st_place_medal:

File: test/api/exclude-members.test.js (hunk 2 above)
Comment (Powersource) [reply test/api/exclude-members.test.js]: wait oops it's supposed to be undefined. but i'll test for that instead

File: test/api/exclude-members.test.js (hunk 3 above)
Comment (Powersource) [reply test/api/exclude-members.test.js]: done now

**Pull request merged**